### DISCUSSION TITLE
Country specific max price

### DISCUSSION THREAD

**Discussion by africantearoa** [jef/streetmerchant#2937] [Ideas and feature requests]
Hi there,

Thanks for the great project.

Is it possible to have country-specific max prices for items (to get around the price differences)?
For example, if I set the max price of something at $1000 USD, it would show Australian stores with prices of ~$1400

If this is not possible, then my suggestion is:
In each store ts file there could be a country variable, default being US
Currency conversion rates are obtained once per day (either manually stored in a file or, more likely, extracted from somewhere online)
Then when we set the max price it can be relative to a particular currency, likely using USD as the reference/default.

Thanks!

**Comment by Mr-Tech-13** [comment]
Thats a great idea, and I think this can be done using cheerio, which may already be used by street merchant.
